docs/js/theme.js

```diff
@@ -0,0 +1,34 @@
+const themesList = ['neuracr', 'dark', 'maua', 'azgheda'];
+const cssVarList = ['primary-color', 'accent-color', 'background-color', 'dark-background-color', 'font-color', 'light-font-color', 'main-font-family', 'title-font-family'];
+const currentThemeStr = localStorage.getItem('currentTheme');
+if (!isNaN(currentThemeStr)) {
+    setTheme(currentThemeStr);
+}
+
+function setTheme(themePosition) {
+    cssVarList.forEach(function(cssVar){
+        document.documentElement.style.setProperty(
+            '--' + cssVar,
+            getComputedStyle(document.documentElement).getPropertyValue(
+                '--' + themesList[themePosition] + '-' + cssVar
+            )
+        );
+    });
+}
+
+function switchTheme() {
+    const currentThemeStr = localStorage.getItem('currentTheme');
+    var currentTheme = 0;
```

Comment:
Tu peux éviter le `var` en faisant
```suggestion
    const nextTheme = !isNaN(currentThemeStr) ? (parseInt(currentThemeStr) + 1) % themesList.length : 0;
```